docs/index.rst

```diff
@@ -0,0 +1,16 @@
+.. sqlalchemy-ibmi documentation master file, created by
+   sphinx-quickstart on Thu Jan 30 10:34:14 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+SQLAlchemy-IBMi
+===============
+
+.. automodule:: sqlalchemy_ibmi.base
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
```

Comment:
I've never seen any benefit of these indices. The search is available in the left side bar and for sqlalchemy-ibmi, you don't really interact with modules at all, you interact with it through the SQLAlchemy interface.
```suggestion
```